DockerSettings.php

```diff
@@ -409,7 +437,11 @@
 //}
 
 ######################### Custom Settings ##########################
-@include( 'CustomSettings.php' );
+if ( file_exists( "$IP/_settings/LocalSettings.php" ) ) {
+	require_once "$IP/_settings/LocalSettings.php";
+} elseif ( file_exists( "$IP/CustomSettings.php" ) ) {
+	require_once "$IP/CustomSettings.php";
+}
```

Comment:
yes, we discussed this. the idea is to mount the `_settings` folder with LocalSettings.php file within to `$IP/_settings` in `docker-compose.yml` like:
```
...
volumes:
    - ./_settings/:/var/www/html/w/_settings
...
```
in this case you can edit the _settings/LocalSettings.php file using any text editor outside  the docker container and this will be edited inside the docker container also and versa vita.

when we edit `CustomSettings.php` in current approach (mounted as `- ./_resources/CustomSettings.php:/var/www/html/w/CustomSettings.php` ) we have to use `nano` text editor because vim or any other text editor  which removes (moves it as a backup copy) the original file and creates a new one with the same name, don't save changes inside the docker container.

Reply:
thanks! this makes sense